stl/inc/ranges

```diff
@@ -981,6 +983,7 @@ namespace ranges {
     };
 
     template <incrementable _Wi>
+        requires integral<_Iota_diff_t<_Wi>> // TRANSITION, as yet unnumbered LWG issue

```

Comment:
This now got a number: LWG-3670

Reply:
Yep - I see the reply from LWGChair in my Inbox. I've asked LWGChair to add this change as the proposed resolution. I didn't in my original submission because I thought we might be able to use `void` for `iterator_traits::difference_type`, return `_Int128` from the pertinent iterator operations, and have `iter_difference_t` use the latter. As it turns out, `iter_difference_t` uses `iterator_traits::difference_type` when `iterator_traits` is specialized, so this won't work. (Which honestly is probably for the best.)